source/_posts/django-vs-flask-which-framework-to-choose.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Django vs Flask - Which framework to choose
+date: 2020-10-02 11:29:00
+author: Himanshu Garg
+tags: 
+	- security
```

Comment:
NAYAN
Traffic monitoring
AI
Livelihood
Driver analytics
Violation mapping
Better commute
Monitor infrastructure defects
Safer roads
Road Safety
Traffic Violation
Artificial Intelligence
India
Smart City
Safety
Data Protection
PARKING DETECTION, 
VEHICLE OCCUPANCY DETECTION
NAYANINDIA
USER INTERACTION
NAYAN IP
NAYAN DATA PROTECTION
Road User
Public Road
Vehicle
Dangers
MACHINE LEARNING

use 3-4 keyword from this list

Reply:
Done